git-fixup

```diff
@@ -210,3 +235,7 @@ if test "$create_commit" == "true"; then
 else
     print_candidates
 fi
+
+if test "$rebase" == "true"; then
```

Comment:
Would it make more sense to move this after the call_commit call above? I don't get the use case of having the plain `git fixup` that outputs a list also call rebase

would need special care to disable rebase from the tab completion scripts too

Reply:
I couldn't follow you with the tab completion stuff. What do you mean?

Reply:
if you have the rebase option enabled through git config and trigger tab completion the script will now end up calling git rebase as part of providing the options

https://github.com/keis/git-fixup/blob/master/completion.zsh#L7

Reply:
Done! Also, looking at `completion.zsh` and `completion.fish`, I see that both `--fixup` and `--base` aren't available to be autocompleted, I don't know if intentionally or not (besides, of course, the newly introduced `--rebase` and `--no-rebase`). I tried fixing those but I still don't know enough about zsh/fish completion to do it myself.

Reply:
no, it's not intentionally left out but I can have a look at getting those up to date after. With the other changes made now I don't think you need to touch those at all.

because now with `--no-commit` and no positional ref arg it also means there will not be a rebase happening, which seems fine to me.

Reply:
oh, that makes a lot of sense